diff/diff_test.go

```diff
@@ -0,0 +1,96 @@
+package diff_test
+
+import (
+	"fmt"
+	"runtime/debug"
+	"strings"
+	"testing"
+
+	"github.com/poy/onpar"
+	"github.com/poy/onpar/diff"
+)
+
+type testStruct struct {
+	Foo string
```

Comment:
I can add a nested struct.  Anonymous structs _should_ function exactly the same, since the reflect package mostly sees them as the same thing (the only difference being the `Anonymous` field on the `StructField` type).